lib/UserInterface/textinput/TextInput.cpp

```diff
@@ -31,6 +31,7 @@
 namespace textinput {
   TextInput::TextInput(Reader& reader, Display& display,
                        const char* HistFile /* = 0 */):
+  fIsDumbTerm(false),
```

Comment:
Isn't that a property of the `TerminalDisplay`? I'd prefer to have the interface there.